node/src/client.rs

```diff
@@ -0,0 +1,352 @@
+//! HydraDX Client abstractions
```

Comment:
Do we need this?

Reply:
I believe it is needed because of the abstraction and generics used in [`service.rs`](https://github.com/galacticcouncil/HydraDX-node/blob/cbb81fb14c5959729a0a22e3a45f1a02c9558757/node/src/service.rs#L305) file.